fitbit/api.py

```diff
@@ -322,17 +198,12 @@ class Fitbit(object):
         'frequent',
     ]
 
-    def __init__(self, client_key, client_secret, oauth2=False, system=US, **kwargs):
+    def __init__(self, client_key, client_secret, system=US, **kwargs):
```

Comment:
@percyperez Since we are already making backward-incompatible changes, what do you think about changing (project wide) `client_key` to`client_id`? That matches better what is shown at dev.fitbit.com